### ISSUE TITLE
Not is compability with androidx

### ISSUE THREAD

**Issue by edrianolima** [sanrixue/videoPlayerWithHeader#4]
...\VideoPlayerPlugin.java:10: error: package android.support.annotation does not exist

No can build project with this error.

My project uses sdk 29 and with androidx on.

**Comment by CEichmueller** [comment]
Yeah this is a common issue. Afaik the author of this package doesn't support this package anymore. But there is a fork which does. He you go: https://github.com/mertcankirtay/videoPlayerWithHeader

**Comment by SethuSenthil** [comment]
FYI here is how you add the fork mentioned by @Letalus to your pubspec.yaml

video_player_header:
  (space) git:
  (spacex2) url: git://github.com/mertcankirtay/videoPlayerWithHeader.git
